frontend/language/src/nb.json

```
@@ -83,6 +83,9 @@
   "contact.email.heading": "Send e-post",
   "contact.email.link": "<0 href=\"mailto:servicedesk@altinn.no\">servicedesk@altinn.no</0>",
   "contact.fetch_app_error_message": "Kunne ikke laste inn tittelen for denne appen. Prøv igjen senere.",
+  "contact.github_issue.content": "Dersom du har behov for funksjonalitet eller ser feil og mangler som må fikses, kan du rapportere det inn til oss ved å opprette en sak i Github.",
+  "contact.github_issue.heading": "Raporter feieler og mangler til oss.",
```

Small typos in the title:
```suggestion
  "contact.github_issue.heading": "Rapporter feil og mangler til oss",
```